options/linux/generic/sys-timerfd.cpp

```diff
@@ -18,10 +18,8 @@ int timerfd_create(int clockid, int flags) {
 
 int timerfd_settime(int fd, int flags, const struct itimerspec *value,
 		struct itimerspec *oldvalue) {
-	__ensure(!oldvalue);
```

Comment:
```
       If  the  old_value argument is not NULL, then the itimerspec structure that it points to is used to return the setting of the timer that was current
       at the time of the call; see the description of timerfd_gettime() following.
```

Why ensure this to NULL?

Reply:
The ensure is getting removed in this PR.